src/marqo/vespa/vespa_client.py

```diff
@@ -237,7 +237,7 @@ def feed_document(self, document: VespaDocument, schema: str, timeout: int = 60)
 
         self._raise_for_status(resp)
```

Comment:
You're still going to fail here so if it's not 200 it won't reach the next line?

Reply:
Yes because we are feeding a single document here so we don't have to catch the error. A HTTP error in this feeding should fail the whole request